internal/rbd/replicationcontrollerserver.go

```diff
@@ -546,6 +551,32 @@ func (rs *ReplicationServer) PromoteVolume(ctx context.Context,
 	return &replication.PromoteVolumeResponse{}, nil
 }
 
+// checkHealthyPrimary checks if the image is in up+stopped state.
```

Comment:
I assume that this function returns no error when the image is Healthy and Primary. How does that relate to `up+stopped`?

Reply:
up+stopped state means the image is healthy and primary, if the image is not in up+stopped state it returns an error message.

Reply:
added a comment :+1: